tests/zones/unit.tests.tst

```diff
@@ -55,3 +55,14 @@ aaaa        600   IN  AAAA  2601:644:500:e210:62f8:1dff:feb8:947a
 ; CNAME Records
 cname       300   IN  CNAME   unit.tests.
 included    300   IN  CNAME   unit.tests.
+
+; SVCB and HTTPS records
+svcb-alias 300 IN SVCB 0 alias-target.unit.test.
+
+svcb-service 300 IN SVCB 1 . ipv4hint=192.0.2.4 port=9000
+svcb-service 300 IN SVCB 2 svcb-target.unit.test. port=9001
+
+https-alias 300 IN HTTPS 0 alias-target.unit.test.
+
+https-service 300 IN HTTPS 1 . ipv4hint=192.0.2.4 port=9000
+https-service 300 IN HTTPS 2 svcb-target.unit.test. port=9001
```

Comment:
Looks like dnspython 2.1.0 added support for these record types https://dnspython.readthedocs.io/en/latest/whatsnew.html#id10 and we already require

https://github.com/octodns/octodns/blob/45753744a66982dd977de3bbe1ddceda07176e42/setup.py#L79

So this should always be supported.

Reply:
What does this mean then, should these lines be removed?